ArduPlane/fence.cpp

```diff
@@ -50,6 +50,10 @@ void Plane::fence_check()
         // user disables/re-enables using the fence channel switch
         return;
     }
+    
+     if( !orig_breaches && new_breaches && plane.is_flying()) {
+        GCS_SEND_TEXT(MAV_SEVERITY_NOTICE, "Fence Breached");
+    }
```

Comment:
```suggestion
     if(new_breaches && plane.is_flying()) {
         GCS_SEND_TEXT(MAV_SEVERITY_NOTICE, "Fence Breached");
     }
```

Copter does not have the `!orig_breaches` in its warning message. Not that I have tested both ways.....

Reply:
I tested this yesterday @IamPete1  . I get one and only one message on each breach.

Reply:
I suspect the difference is if you breach multiple fences. As you have it it will not warn twice if you were to breach a polygon fence and then a altitude fence.

I think my proposed change (copied from copter), would warn twice in that case?

I think warning per breach of each different type of fence is correct, it is also what copter does and there is value in keeping this uniform between vehicles.